specs/user_status.json

```diff
@@ -192,11 +191,7 @@
                 "$ref": "#/components/schemas/OCSMeta"
               },
               "data": {
-                "anyOf": [
-                  {
-                    "$ref": "#/components/schemas/Status"
-                  }
-                ]
+                "$ref": "#/components/schemas/Status"
```

Comment:
This might fail, since an empty array can be returned here if the user has no status. Maybe just go for the `[oneOf: [array, Status]]` thing?